components/scientific/hdf5/hdf5.p5m

```diff
@@ -163,54 +165,52 @@ file path=usr/include/hdf5/h5s.mod
 file path=usr/include/hdf5/h5t.mod
 file path=usr/include/hdf5/h5t_provisional.mod
 file path=usr/include/hdf5/h5tb.mod
-file path=usr/include/hdf5/h5test_kind_sizeof_mod.mod
+file path=usr/include/hdf5/h5test_kind_storage_size_mod.mod
 file path=usr/include/hdf5/h5z.mod
 file path=usr/include/hdf5/hdf5.h
 file path=usr/include/hdf5/hdf5.mod
 file path=usr/include/hdf5/hdf5_hl.h
 file path=usr/lib/$(MACH64)/libhdf5.settings
-link path=usr/lib/$(MACH64)/libhdf5.so target=libhdf5.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5.so.8 target=libhdf5.so.8.0.2
-file path=usr/lib/$(MACH64)/libhdf5.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_cpp.so target=libhdf5_cpp.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_cpp.so.8 target=libhdf5_cpp.so.8.0.2
-file path=usr/lib/$(MACH64)/libhdf5_cpp.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_fortran.so \
-    target=libhdf5_fortran.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_fortran.so.8 \
-    target=libhdf5_fortran.so.8.0.2
-file path=usr/lib/$(MACH64)/libhdf5_fortran.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_hl.so target=libhdf5_hl.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_hl.so.8 target=libhdf5_hl.so.8.0.2
-file path=usr/lib/$(MACH64)/libhdf5_hl.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_hl_cpp.so target=libhdf5_hl_cpp.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5_hl_cpp.so.8 \
-    target=libhdf5_hl_cpp.so.8.0.2
-file path=usr/lib/$(MACH64)/libhdf5_hl_cpp.so.8.0.2
+link path=usr/lib/$(MACH64)/libhdf5.so target=libhdf5.so.10.1.0
+link path=usr/lib/$(MACH64)/libhdf5.so.10 target=libhdf5.so.10.1.0
+file path=usr/lib/$(MACH64)/libhdf5.so.10.1.0
+link path=usr/lib/$(MACH64)/libhdf5_cpp.so target=libhdf5_cpp.so.11.0.0
+link path=usr/lib/$(MACH64)/libhdf5_cpp.so.11 target=libhdf5_cpp.so.11.0.0
+file path=usr/lib/$(MACH64)/libhdf5_cpp.so.11.0.0
+link path=usr/lib/$(MACH64)/libhdf5_fortran.so target=libhdf5_fortran.so.10.0.2
+link path=usr/lib/$(MACH64)/libhdf5_fortran.so.10 \
+    target=libhdf5_fortran.so.10.0.2
+file path=usr/lib/$(MACH64)/libhdf5_fortran.so.10.0.2
+link path=usr/lib/$(MACH64)/libhdf5_hl.so target=libhdf5_hl.so.10.0.2
+link path=usr/lib/$(MACH64)/libhdf5_hl.so.10 target=libhdf5_hl.so.10.0.2
+file path=usr/lib/$(MACH64)/libhdf5_hl.so.10.0.2
+link path=usr/lib/$(MACH64)/libhdf5_hl_cpp.so target=libhdf5_hl_cpp.so.11.0.0
+link path=usr/lib/$(MACH64)/libhdf5_hl_cpp.so.11 target=libhdf5_hl_cpp.so.11.0.0
+file path=usr/lib/$(MACH64)/libhdf5_hl_cpp.so.11.0.0
 link path=usr/lib/$(MACH64)/libhdf5hl_fortran.so \
-    target=libhdf5hl_fortran.so.8.0.2
-link path=usr/lib/$(MACH64)/libhdf5hl_fortran.so.8 \
-    target=libhdf5hl_fortran.so.8.0.2
-file path=usr/lib/$(MACH64)/libhdf5hl_fortran.so.8.0.2
+    target=libhdf5hl_fortran.so.10.0.2
+link path=usr/lib/$(MACH64)/libhdf5hl_fortran.so.10 \
+    target=libhdf5hl_fortran.so.10.0.2
+file path=usr/lib/$(MACH64)/libhdf5hl_fortran.so.10.0.2
 file path=usr/lib/libhdf5.settings
-link path=usr/lib/libhdf5.so target=libhdf5.so.8.0.2
```

Comment:
Not to my knowledge as I only use it for non-packaged software but it is worth checking.
I do not have OI running at home for now though, so I'll do it tomorrow.